src/init-ceph.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
unlike `getopts`, `getopt` is an external command. so maybe we don't need it at all? and imho, as a startup script, it would be ideal to be POSIX compliant, and not rely on a certain shell implementation.